test/support/controller/undocumented_controller.ex

```diff
@@ -1,4 +1,4 @@
-defmodule BlueBird.Test.Support.TestControllerUndocumented do
+defmodule BlueBird.Test.Support.TestUndocumentedController do
```

Comment:
`BlueBird.Test.Support.TestController`, `BlueBird.Test.Support.TestNamedController` and `BlueBird.Test.Support.TestUndocumentedController` is a bit redundant. Can we rename them to `BlueBird.Test.Support.Controller` etc.?

Reply:
Yes... sure.